modules/aws_account_permissions/variables.tf

```diff
@@ -0,0 +1,32 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# Required PARAMETERS
+# These variables have default values and can be modified based on the requirement.
+# ---------------------------------------------------------------------------------------------------------------------
+
+
```

Comment:
I find this comment confusing.
How can a variable be both required and a default value?
 ```suggestion
```